### PR TITLE
Add logs for swap and sign method

### PR DESCRIPTION
**Pull request by yury-dubinin**

### CODE REVIEW

File: packages/e2e/tests/swap.usdc.wallet.spec.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ test.describe("Test Swap to/from USDC feature", () => {
     await tradePage.selectPair("OSMO", "USDC");
     await tradePage.enterAmount("0.2");
     await tradePage.showSwapInfo();
-    const { msgContentAmount } = await tradePage.swapAndGetWalletMsg(context);
+    const msgContentAmount = await tradePage.swapAndGetWalletMsg(context);
```
[the remark this answers is not in the document]
Comment (yury-dubinin) [reply packages/e2e/tests/swap.usdc.wallet.spec.ts]: applied.

File: packages/e2e/tests/monitoring.market.wallet.spec.ts [hunk 2]
```diff
@@ -76,9 +74,7 @@ test.describe("Test Market Buy/Sell Order feature", () => {
     await tradePage.enterAmount("1.54");
     await tradePage.isSufficientBalanceForTrade();
     await tradePage.showSwapInfo();
-    const { msgContentAmount } = await tradePage.sellAndGetWalletMsg(context);
-    expect(msgContentAmount).toBeTruthy();
-    expect(msgContentAmount).toContain("type: osmosis/poolmanager/");
+    await tradePage.buyAndApprove(context);
```
[the remark this answers is not in the document]
Comment (yury-dubinin) [reply packages/e2e/tests/monitoring.market.wallet.spec.ts:77]: good catch!

File: packages/e2e/tests/swap.usdc.wallet.spec.ts [hunk 3]
```diff
@@ -1,194 +1,184 @@
...
+    await tradePage.selectPair("AKT", "USDC");
+    await tradePage.enterAmount("0.025");
+    await tradePage.showSwapInfo();
+    await tradePage.swapAndApprove(context);
+    //expect(msgContent).toContain(`denom: ${AKT}`);
+    //expect(msgContent).toContain(`sender: ${walletId}`);
+    //expect(msgContent).toContain(`token_out_denom: ${USDC}`);
+    await tradePage.isTransactionSuccesful();
+    await tradePage.getTransactionUrl();
+  });
```
[the remark this answers is not in the document]
Comment (yury-dubinin) [reply packages/e2e/tests/swap.usdc.wallet.spec.ts:183]: will do in next pull request, thanks.

File: packages/e2e/tests/swap.usdc.wallet.spec.ts [hunk 4]
```diff
@@ -2,193 +1,184 @@
...
+  test("User should be able to swap OSMO to USDC", async () => {
+    await tradePage.goto();
+    await tradePage.selectPair("OSMO", "USDC");
+    await tradePage.enterAmount("0.2");
+    await tradePage.showSwapInfo();
+    await tradePage.swapAndApprove(context);
+    //expect(msgContent).toContain(`token_out_denom: ${USDC}`);
+    //expect(msgContent).toContain(`sender: ${walletId}`);
+    //expect(msgContent).toContain("denom: uosmo");
+    await tradePage.isTransactionSuccesful();
```
[the remark this answers is not in the document]
Comment (yury-dubinin) [reply packages/e2e/tests/swap.usdc.wallet.spec.ts:64]: no need for this.

**Review by DavideSegullo — APPROVED** [review]

**Pull request merged**